doc/options.yaml

```diff
@@ -1202,8 +1213,9 @@ useMatrixFreedrdw:
   desc: >
     Flag that determines if ADflow will use the matrix-free routines to compute mat-vec products with the full Jacobian.
     If False, ADflow will form the full matrix exactly and mat-vec multiplications will be matrix-based.
+    This should always be True for overset meshes, otherwise the gradients will be inaccurate.
```

Comment:
maybe mention this is an unresolved bug and link to an issue on github? There's no inherent reason why a matrix-based adjoint should not work for overset meshes.